packages/base-styles/index.js

```diff
@@ -0,0 +1 @@
+// index.js
```

Comment:
One more thing before we land. Can we update the comment to be more informative to ensure it isn't deleted in the future?

Reply:
Coming right up!